src/components/Scheme/ContributionRewardExtRewarders/Competition/StatusBlob.tsx

```diff
@@ -0,0 +1,22 @@
+import { ICompetitionProposalState, ICompetitionSuggestionState } from "@daostack/client";
```

Comment:
what is a StatusBlob?

Reply:
added:

/**
 * displays the current status of a competition
 */